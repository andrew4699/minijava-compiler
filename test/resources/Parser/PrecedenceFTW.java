class P {
  public static void main(String[] args) {
    System.out.println(1);
  }
}
class FTW {
  public int p_ftw() {
    return 1 + 2 * 3 + 4 + 4 * 5 < 1 + 2 * 5 && !true;
  }
}