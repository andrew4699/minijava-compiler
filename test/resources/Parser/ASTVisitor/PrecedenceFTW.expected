Program
  MainClass P (line 1)
    Print (line 3)
      1
  Class FTW (line 6)
    MethodDecl p_ftw (line 7)
      returns int
      Return (((((1 + (2 * 3)) + 4) + (4 * 5)) < (1 + (2 * 5))) && !true) (line 8)
